plugins/Login/lang/en.json

```diff
@@ -1,68 +1,68 @@
+        "PluginDescription": "Provides username and password login as well as password reset functionality. The login method can be changed by using another login plugin such as LoginLdap available on the marketplace.",
         "RememberMe": "Remember Me",
-        "SuspiciousLoginAttemptsInLastHourEmailSubject": "We've noticed some suspicious login attempts to your account...",
-        "SuspiciousLoginAttemptsInLastHourEmail1": "We've noticed a suspiciously high number of login attempts to your Matomo account in the last hour, specifically %1$s login attempts from %2$s distinct IP addresses. We suspect someone may be trying to break into your account.",
-        "SuspiciousLoginAttemptsInLastHourEmail2": "In order to safeguard your Matomo, we recommend taking the following precautions:",
-        "SuspiciousLoginAttemptsInLastHourEmail3": "Ensure your password is a secure, random password of an appropriate length (for example, at least 30 characters).",
-        "SuspiciousLoginAttemptsInLastHourEmail4": "Setup two-factor auth so attackers will need more information than just your password in order to login.",
-        "SuspiciousLoginAttemptsInLastHourEmail5": "Additionally, if your Matomo has a limited set of users or IPs through which users will access it, it may be beneficial to setup a IP address allowlist. %1$sRead our docs for more information.%2$s",
-        "LoginNotAllowedBecauseUserLoginBlocked": "Login functionality is temporarily disabled since we've a suspicious amount of failed login attempts in the last hour.",
-        "InvitationTitle": "Accept Invitation",
-        "InvitationDeclineTitle": "Decline Invitation",
-        "InvitationDeclineBody": "Are you sure you want to decline this Invitation?",
-        "InvitationHints": "(to cancel this action, just leave the page)",
+        "SuspiciousLoginAttemptsInLastHourEmailSubject": "Some suspicious login attempts were made using your username…",
+        "SuspiciousLoginAttemptsInLastHourEmail1": "A suspiciously high number of login attempts was made on your Matomo account the last hour. Specifically %1$s login attempts from %2$s distinct IP addresses. Someone may be trying to break into your account.",
+        "SuspiciousLoginAttemptsInLastHourEmail2": "Do the following:",
+        "SuspiciousLoginAttemptsInLastHourEmail3": "Ensure your password is a secure, random password of at least 30 characters.",
+        "SuspiciousLoginAttemptsInLastHourEmail4": "Set up two-factor authentication so attackers need more than just your password to log in.",
+        "SuspiciousLoginAttemptsInLastHourEmail5": "Set up a list of IP addresses to never block if your Matomo has a limited set of users or IPs users access it from. %1$sRead the docs for more info.%2$s",
+        "LoginNotAllowedBecauseUserLoginBlocked": "Logging in has been turned off since a suspicious amount of failed attempts were made the last hour.",
```

Comment:
```suggestion
        "LoginNotAllowedBecauseUserLoginBlocked": "Logging in has been turned off since a suspicious amount of failed attempts were made during the last hour.",
```